dev/com.ibm.ws.jaxws.clientcontainer_fat/fat/src/com/ibm/ws/jaxws/clientcontainer/fat/FATSuite.java

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws.clientcontainer.fat;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import componenttest.custom.junit.runner.AlwaysPassesTest;
+
+@RunWith(Suite.class)
+@SuiteClasses({
+
+                AlwaysPassesTest.class,
+                FATSuiteWithJava7.class
```

Comment:
delete this line and add the classes from FATSuiteWithJava7 directly

Reply:
Since we are not limiting Java version to Java 7 only, I guess it would be good to remove AlwaysPassesTest.class, right? Do you see any reason to keep it?